nornir/core/inventory.py

```diff
@@ -196,42 +196,52 @@ def nornir(self, value):
             self._nornir = value
 
     @property
-    def host(self):
-        """String used to connect to the device. Either ``nornir_host`` or ``self.name``"""
-        return self.get("nornir_host", self.name)
+    def hostname(self):
+        """String used to connect to the device. Either ``hostname`` or ``self.name``"""
+        return self.get("hostname", self.name)
+
```

Comment:
I think we agreed on `host`, no?

Reply:
I don't want to use host because `host` usually refers to the `Host` object so it's confusing, ie, `task.host.host` vs `task.host.hostname`. I am also fine naming it something else, I'd just avoid `host`.

Reply:
I think `host` makes sense in the way that it can refer both to an IP address and a hostname. Though I agree that `task.host.host` looks a bit strange.

Reply:
I am also fine with naming it something else, but what? :) I have no other ideas.
The reason I don't like `hostname` is because it can be also an IP address. IP address is not a hostname.
The reason I prefer `host` is not because I like it, but because it is used by majority of libraries:
netmiko uses `host` and `ip`
pyvmomi uses `host`
ncclient uses `host`
napalm uses `hostname`
django uses `ALLOWED_HOSTS` 

While `task.host.host` does not look very beautiful, I would still prefer this.

Reply:
I am fine with `address`, `ip`, `connection_address`, `mgmt_ip`,  `quantum_connection`, others but I am going to veto `host` for consistency purposes :D

Reply:
I vote for `address`

Reply:
I vote `host` or `hostname`. I pretty strongly dislike address.

I am fine with `hostname`. I would say we just leave it as that and move on.